### PR TITLE
fix: Add null checks for baseOptions (GH-6447)

### PR DESCRIPTION
**Pull request by bgambocjaviniar**
Closes GH-6447

### CODE REVIEW

File: projects/storefrontlib/src/cms-components/cart/cart-shared/cart-item/cart-item.component.html [hunk 1]
```diff
@@ -38,15 +38,17 @@
...
-          <div
-            *ngFor="
-              let variant of item.product.baseOptions[0]?.selected
-                ?.variantOptionQualifiers
-            "
-            class="cx-property"
-          >
-            <div class="cx-label" *ngIf="variant.name && variant.value">
-              {{ variant.name }}: {{ variant.value }}
+          <div *ngIf="item.product.baseOptions.length">
```
Comment (mirkosoft) [inline projects/storefrontlib/src/cms-components/cart/cart-shared/cart-item/cart-item.component.html]: Using ng-container won't produce additional div

File: projects/storefrontlib/src/cms-components/myaccount/order/amend-order/amend-order-items/amend-order-items.component.html [hunk 2]
```diff
@@ -62,18 +62,20 @@
...
-                    ?.variantOptionQualifiers
-                "
-                class="cx-property"
-              >
-                <div class="cx-label" *ngIf="variant.name">
-                  {{ variant.name }}:
-                </div>
-                <div class="cx-value" *ngIf="variant.value">
-                  {{ variant.value }}
+              <div *ngIf="item.product.baseOptions.length">
```
Comment (mirkosoft) [inline projects/storefrontlib/src/cms-components/myaccount/order/amend-order/amend-order-items/amend-order-items.component.html]: Using ng-container won't produce additional div

File: projects/storefrontlib/src/cms-components/wish-list/components/wish-list-item/wish-list-item.component.html [hunk 3]
```diff
@@ -26,15 +26,17 @@
...
-        <div
-          *ngFor="
-            let variant of cartEntry.product.baseOptions[0]?.selected
-              ?.variantOptionQualifiers
-          "
-          class="cx-property"
-        >
-          <div class="cx-label" *ngIf="variant.name && variant.value">
-            {{ variant.name }}: {{ variant.value }}
+        <div *ngIf="cartEntry.product.baseOptions.length">
```
Comment (mirkosoft) [inline projects/storefrontlib/src/cms-components/wish-list/components/wish-list-item/wish-list-item.component.html]: Using ng-container won't produce additional div

File: projects/storefrontlib/src/cms-components/cart/cart-shared/cart-item/cart-item.component.html (hunk 1 above)
Comment (bgambocjaviniar) [reply projects/storefrontlib/src/cms-components/cart/cart-shared/cart-item/cart-item.component.html]: done

File: projects/storefrontlib/src/cms-components/myaccount/order/amend-order/amend-order-items/amend-order-items.component.html (hunk 2 above)
Comment (bgambocjaviniar) [reply projects/storefrontlib/src/cms-components/myaccount/order/amend-order/amend-order-items/amend-order-items.component.html]: done

File: projects/storefrontlib/src/cms-components/wish-list/components/wish-list-item/wish-list-item.component.html (hunk 3 above)
Comment (bgambocjaviniar) [reply projects/storefrontlib/src/cms-components/wish-list/components/wish-list-item/wish-list-item.component.html]: done

**Pull request merged**